src/components/Settings/AdvancedSettings.tsx

```diff
@@ -0,0 +1,49 @@
+import { TextInput, Group, Button } from '@mantine/core';
+import { useState } from 'react';
+import { useConfig } from '../../tools/state';
+
+export default function TitleChanger() {
+  const { config, loadConfig, setConfig, getConfigs } = useConfig();
+  const [customTitle, setCustomTitle] = useState(config.title);
+  const [customLogo, setCustomLogo] = useState(config.logo);
+  const [customFavicon, setCustomFavicon] = useState(config.favicon);
+
+  const saveChanges = () => {
+    setConfig({
+      ...config,
+      title: customTitle || "Homarr 🦞",
+      logo: customLogo || "/imgs/logo.png",
+      favicon: customFavicon || "/favicon.svg",
```

Comment:
Remove the default values so that it doesn't save new entries unless the user has specified a changed the value in the inputs.